src/bvers.f.in

```diff
@@ -1,13 +1,16 @@
 C> @file
-C> @author J. Ator
-C> @date 2020-10-21
+C> @brief Return a character string containing the version
+C> number of the BUFRLIB software.
 
-C> @brief This subroutine returns a character string containing
-C> the version number of the BUFR archive library software.
+C> @author J. Ator
```

Comment:
I think the brief sentance for the subroutine has to come first.

Reply:
OK, I'll fix this.

Reply:
Also I believe we found that we don't need the brief tag in the subroutine description, but we do need it in the file description.

Reply:
Agreed, and right now it's only in the file block.  You may just be confused by the + and - labels in the diff output.